numba/cuda/compiler.py

```diff
@@ -477,6 +521,27 @@ def _prepare_args(self, ty, val, stream, retr, kernelargs):
             raise NotImplementedError(ty, val)
 
 
+    @property
+    def autotune(self):
```

Comment:
Done. (It's minimal as it isn't clear to what degree the API should be considered 'public'.